molecule/collectd_connection/verify.yml

```diff
@@ -30,6 +30,7 @@
       command:
         cat {{ collectd_conf_output_dir }}/network.conf
       register: conf
+      changed_when: True
```

Comment:
I couldn't figure out why this one in particular is set to True. Does anything care about the changed status of this check?

Reply:
That shouldn't be set to True

Reply:
Glad I could be useful!